CIP-????/README.md

```diff
@@ -0,0 +1,285 @@
+![image](https://user-images.githubusercontent.com/47434720/206574191-22aa490a-5870-4853-906b-443284458987.png)
+![image](https://user-images.githubusercontent.com/47434720/206574354-5dd81551-efc6-4f69-a2aa-282bb40e5084.png)
+
+
+&nbsp;<p>
+
+# Rationale
+
+This design is simple, so many tools on the cardano blockchain can adopt it easily and a few have already started to implement it.
+The original CIP-0020 design allowed the addition of new entries like the `"enc":` key for encrypted messages in this CIP.
+There is also for example [CIP-8](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0008), but CIP-8 doesn't really fulfill the simplicity of just providing encrypted messages. CIP-8 is focused on Signing, which is not needed for encryption. The method to generate encrypted messages here is not intended to verify the owner of a message via signing. There is no need that everything on Cardano must be difficult. Also using such CBOR encoded structures would break all currently implemented transaction message solutions. This CIP uses openssl and base64 encoding, and endusers could even copy&paste such text into other tools, etc. 
+
+## Implementation suggestions
+ 
+Wallets/Tools can implement an autodecryption attempt with the default passphrase on such messages, to give the user a more streamlined experience. The communication should be done via https or similar to make sure the message cleartext is not exposed again during the transmission.
+Additionally the Tools can prompt for an input and decrypt the message once the user has requested it, this decryption should be done on the user frontend for further security.
+
+## Handling ill-formed 674 metadata ##
+
+Like with CIP-0020, it is up to the wallet-/display-/receiver-implementor to parse and check the provided metadata. As for the current state, its not possible to have the same label "674" more than once in a cardano transaction. So a check about that can be ignored at the moment. This CIP provides the correct implementation format, the parsing should search for the "674" metadata label, the "msg" and the "enc" key underneath it. There should also be a check, that the provided data within that "msg" key is an array. All other implementations like a missing "msg" key, or a single string instead of an array, should be marked by the display-implementor as "invalid". Additional keys within the "674" label should not affect the parsing of the "msg" and the "enc" key. As written above, we will likely see more entries here in the future like a "version" key for example, so additional keys should not harm the parsing of the "msg" and "enc" key. 
+
+## Implementation conclusion ##
+
+An encrypted transaction message should be considered valid if the following apply:
+
+* Label = 674.
+* has property "enc".
+* enc property contains a supported method like `basic`
+* has property "msg".
+* msg property contains an array of strings, even for a single-line message.
```

Comment:
This (as well as the base64 encoding of each string) sounds like a bit of a unfortunate design decision? On-chain, metadata are actually encoded as bytes using CBOR; which means that each message can actually be encoded as a plain byte-array, saving 33% size over the base64 encoding. 

If anything, I'd at least like to see this design decision (choosing UTF-8 text string over plain bytes) explained in the rationale section.

Reply:
Comment was added about that in the rational

Reply:
I am now pleased with CIP editor's hat; but not with my technical expert's one 😬. I am not convinced by the justification of using base64 and I would strongly advocate against that.